Cargo.toml

```diff
@@ -19,68 +19,69 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 license-file = "LICENSE-APACHE"
 
-
 [workspace.dependencies]
-anyhow = "1"
-arbitrary = { version = "1", features = ["derive"] }
-arbtest = "0.2"
-async-stm = "0.2"
-async-trait = "0.1"
-axum = { version = "0.6", features = ["ws"] }
-base64 = "0.21"
-blake2b_simd = "1.0"
-bytes = "1.4"
-clap = { version = "4.1", features = ["derive", "env"] }
-config = "0.13"
-dirs = "5.0"
-ethers = { version = "2.0", features = ["abigen", "ws"] }
-ethers-core = { version = "2.0" }
-fnv = "1.0"
-futures = "0.3"
-hex = "0.4"
-jsonrpc-v2 = { version = "0.11", default-features = false, features = ["bytes-v10"] }
-k256 = "0.11"                                                                         # Same as tendermint-rs
-lazy_static = "1.4"
-libsecp256k1 = "0.7"
-multihash = { version = "0.16.1", default-features = false }
-num-traits = "0.2"
-paste = "1"
-prost = { version = "0.11" }
-quickcheck = "1"
-quickcheck_macros = "1"
-rand = "0.8"
-rand_chacha = "0.3"
-regex = "1"
-serde = { version = "1", features = ["derive"] }
-serde_json = { version = "1" }
-serde_tuple = "0.5"
-serde_with = "2.3"
-tempfile = "3.3"
-thiserror = "1"
-tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
-tracing = "0.1"
-tracing-subscriber = "0.3"
-
+anyhow = "1" 
+arbitrary = {version = "1", features = ["derive"]} 
```

Comment:
It looks like your tom formatter removes extra spaces. I wonder if we'll have a constant churn on this.

Reply:
I disabled the toml formatter to avoid this from happening again. Sorry about that.

Reply:
Cheers! I also use a TOML formatter but I couldnt' find a whitespace setting for this one, only arrays and such.